google/resource_compute_url_map_test.go

```diff
@@ -320,3 +347,40 @@ resource "google_compute_url_map" "foobar" {
     }
 }
 `, acctest.RandString(10), acctest.RandString(10), acctest.RandString(10))
+
+func testAccComputeUrlMap_noPathRules(bsName, hcName, umName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_backend_service" "foobar" {
```

Comment:
Can you update the indentation and line up the spacing in these resources?
i.e. for this one:
```
resource "google_compute_backend_service" "foobar" {
    name          = "urlmap-test-%s"
    health_checks = ["${google_compute_http_health_check.zero.self_link}"]
}